common/static/common/js/spec/discussion/view/discussion_thread_list_view_spec.js

```diff
@@ -233,6 +233,19 @@
             });
         });
 
+        it('filter should clear search alert when alternate term was searched', function() {
```

Comment:
@Rabia23 I have added a sperate test in `DiscussionThreadListView`, we don't need to define spec suit because it has only one test